mobile/native/src/trade/position/mod.rs

```diff
@@ -0,0 +1,58 @@
+use crate::trade::ContractSymbolTrade;
+use crate::trade::DirectionTrade;
+pub mod handler;
+pub mod subscriber;
+
+#[derive(Debug, Clone)]
+pub enum PositionStateTrade {
+    /// The position is open
+    ///
+    /// Open in the sense, that there is an active position that is being rolled-over.
+    /// Note that a "closed" position does not exist, but is just removed.
+    /// During the process of getting closed (after creating the counter-order that will wipe out
+    /// the position), the position is in state "Closing".
+    ///
+    /// Transitions:
+    /// Open->Closing
+    Open,
+    /// The position is in the process of being closed
+    ///
+    /// The user has created an order that will wipe out the position.
+    /// Once this order has been filled the "closed" the position is not shown in the user
+    /// interface, so we don't have a "closed" state because no position data will be provided to
+    /// the user interface.
+    Closing,
+}
+
+#[derive(Debug, Clone)]
+pub struct PositionTrade {
```

Comment:
We might want to add an identifier for the position in question. This is relevant once we have multiple positions (e.g. ETH/USD, BTC/USD).

Reply:
Depending on how we want to handle the lifecycle the `ContractSymbol` might be enough.

For one `ContractSymbol` there can only be one positions at any time.

At least from the UI perspective a `Position` is only displayed as long as it is open (with the caveat of also displaying it in out `closing` state, but that's basically still `open` - we just track that there is an order that wants to close it in the state).

Internally we *could* give each position an identifier and also represent a `closed` state. I'm not sure about that yet and would decide once we introduce the DAL.

Reply:
> For one contract symbol there can only be one positions at any time.

Ah right, way easier :) 👍